tools/packaging/static-build/cache_components.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+# Copyright (c) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+source "${script_dir}/../scripts/lib.sh"
+source "${script_dir}/qemu/build-static-qemu-cc.sh"
+
+export qemu_cc_tarball_name="kata-static-qemu-cc.tar.gz"
```

Comment:
At some point we will need caching on the `main` branch too, and we will need to use switches to decide if it is a cache for CC or not. Other scripts on the CCv0 branch have used the `KATA_BUILD_CC` to take a decision. For example, this line could be:

```sh
if [ "$KATA_BUILD_CC}" == "yes" ]; then
    export qemu_tarball_name="kata-static-qemu-cc.tar.gz"
else
    export qemu_tarball_name="kata-static-qemu-cc.tar.gz"
fi
```
The algorithm to generate the cache is generic, so the "cc" reference can be removed from the methods and variables names (e.g. `cache_qemu_cc_artifacts()` -> `cache_qemu_artifacts()`).

Or am I missing something here? :)

Reply:
@wainersm thanks for the info I did not know about the KATA_BUILD_CC, and about the name for me it works ok to remove or not I just followed the current names like `build-static-qemu-cc.sh` that you have